scanner/src/funTest/kotlin/scanners/ScannerIntegrationFunTest.kt

```diff
@@ -58,22 +93,28 @@ class ScannerIntegrationFunTest : StringSpec() {
         }
     }
 
-    class DummyScanner(
-        name: String,
-        scannerConfig: ScannerConfiguration,
-        downloaderConfig: DownloaderConfiguration
-    ) : PathScanner(name, scannerConfig, downloaderConfig) {
-        override val version = "1.0"
-        override val configuration = ""
+    class DummyScanner : PathScannerWrapper {
+        override val name = "Dummy"
+        override val details = ScannerDetails(name = name, version = "1.0.0", configuration = "")
+        override val criteria: ScannerCriteria = ScannerCriteria.forDetails(details)
 
-        override fun scanPathInternal(path: File): ScanSummary {
+        override fun filterSecretOptions(options: Options): Options = options
+
+        override fun scanPath(path: File, context: ScanContext): ScanSummary {
             val time = Instant.now()
 
+            val licenseFindings = path.listFiles().orEmpty().mapTo(sortedSetOf()) { file ->
+                LicenseFinding(
+                    license = SpdxConstants.NONE,
+                    location = TextLocation(file.relativeTo(path).invariantSeparatorsPath, TextLocation.UNKNOWN_LINE)
+                )
+            }
+
             return ScanSummary(
                 startTime = time,
                 endTime = time,
-                packageVerificationCode = calculatePackageVerificationCode(path),
-                licenseFindings = sortedSetOf(),
+                packageVerificationCode = "",
```

Comment:
> As the experimental scanner does not support the calculation of the package verification code, create a list of fake license findings instead to verify that the correct source code was scanned.

I'm not sure I can follow you here: Couldn't you just have the left the call to `calculatePackageVerificationCode()` in here?

Reply:
No, the experimental scanner discards any package verification codes from the scan summaries returned by the scanners, so providing one here would be useless. Removing the package verification code from the model is one of the cleanup tasks for after this PR, and they will only be available again once #5681 is implemented.